packages/devtools_app/lib/src/screens/deep_link_validation/deep_link_list_view.dart

```diff
@@ -248,61 +252,64 @@ class _DeepLinkListViewTopPanel extends StatelessWidget {
       includeBottomBorder: false,
       tall: true,
       title: Row(
-        mainAxisAlignment: MainAxisAlignment.spaceBetween,
         children: [
           Text(
             'Validate and fix',
             style: Theme.of(context).textTheme.titleSmall,
           ),
-          ValueListenableBuilder(
-            valueListenable: controller.selectedVariantIndex,
-            builder: (_, value, __) {
-              return _AndroidVariantDropdown(
-                androidVariants:
-                    controller.selectedProject.value!.androidVariants,
-                index: value,
-                onVariantIndexSelected: (index) {
-                  controller.selectedVariantIndex.value = index;
-                },
-              );
-            },
+          const Spacer(),
+          _VariantDropdown(
+            os: PlatformOS.android,
+            controller: controller,
+          ),
+          _VariantDropdown(
```

Comment:
If we are connected to a running application, would it make more sense to only show whichever of these is relevant for the connected app platform instead of both?

Reply:
If we only have only one dropdown here, we should also run checks and show result for only one platform.

I think it's more consistent to always display the check result for both platform, but will discuss this with team today for sure.

if a domain is detected on both Android and iOS, the connected running app is on Android, and the result table displayed as :
[Domain]                        [OS]
[example.com]               [Android]

I might wonder why iOS is not here